src/index.cpp

```diff
@@ -785,6 +788,7 @@ size_t Index<T, TagT, LabelT>::load_graph(std::string filename, size_t expected_
     }
 #endif
 
+    _memory_in_bytes += cc * sizeof(uint32_t);
```

Comment:
Are we sure that the allocation is exactly this at all times. Perhaps it starts with this, but it also changes as the graph is edited, right?

Reply:
for our case, our graph is fixed

Reply:
we want to know the memory usage for each index